### ISSUE TITLE
Discriminator loss function

### ISSUE THREAD

**Issue by baumgach** [shekkizh/WassersteinGAN.tensorflow#12]
I don't understand how exactly the loss function in line 5 of algorithm 1 in the original WGAN paper is implemented here. In your code you **minimise**

`self.discriminator_loss = discriminator_loss_fake + discriminator_loss_real`

However, according to the paper shouldn't it be **maximising**:

`self.discriminator_loss = discriminator_loss_real - discriminator_loss_fake`

or alternatively **minimising**:

`self.discriminator_loss = discriminator_loss_fake - discriminator_loss_real`

That is, should this be a minus in your total loss?

**Comment by haleqiu** [comment]
I thought the first algorithm is implemented for cross entropy, which 
`discriminator_loss_fake + discriminator_loss_real`
